airbyte-webapp/src/core/domain/connection/WebBackendConnectionService.ts

```diff
@@ -4,7 +4,7 @@ import {
   webBackendCreateConnection,
   webBackendGetConnection,
   webBackendListConnectionsForWorkspace,
-  webBackendUpdateConnection,
+  webBackendUpdateConnectionNew as webBackendUpdateConnection,
```

Comment:
Yes it is; this endpoint was created so that we could merge backend changes related to this new flow in before the frontend changes were merged. We have a ticket here to remove this temporary endpoint after the launch of per-stream: https://github.com/airbytehq/airbyte/issues/14733